application/dash_application/views/preqc_rna.py

```diff
@@ -212,7 +212,7 @@ def generate_reads_per_start_point(df, colour_by, shape_by, cutoff):
         df,
         lambda d: d[PINERY_COL.SampleName],
         lambda d: d[RNA_COL.ReadsPerStartPoint],
-        "",
+        "Fraction",
```

Comment:
Can the RPSP Y axis not be labelled, rather than call it "Fraction"? As you said, it's :woman_shrugging: as to why they call it that

Reply:
Done